galaxy/templates/deployment-job.yaml

```diff
@@ -44,6 +44,34 @@ spec:
               subPath: config
             - name: galaxy-data
               mountPath: {{.Values.persistence.mountPath}}
+        - name: {{ .Chart.Name }}-initscripts
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          command: ['sh', '-c', 'echo "Starting to run init scripts"
```

Comment:
Most other charts seem to map initScripts as configMaps (also postgresql and keycloak). I think this makes sense? https://github.com/helm/charts/search?q=initscripts&unscoped_q=initscripts

Reply:
They are mapped as configmaps to inject into the containers, but run through an init container.
Are you saying we should just map them and not run them, or run them in the main container before the command? (that would make their effects more persistent imo, as opposed to how it is now where they are only persistent if they change something on the persistent disk)

Reply:
Changed to use configmaps (or secrets) to inject, and run in the main containers before the command. Only `sh` scripts supported for now.